src/web/mjs/connectors/KumaManga.mjs

```diff
@@ -0,0 +1,39 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class KumaManga extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'kumamanga';
+        super.label = 'KumaManga';
+        this.tags = [ 'manga', 'indonesian' ];
+        this.url = 'https://kumamanga.my.id';
+        this.path = '/manga-list/?list';
+
+        this.querMangaTitleFromURI = 'div.series-title h2';
+        this.queryMangas = 'div.mangalist-blc ul li a';
+        this.queryChapters = 'div.series-chapter ul.series-chapterlist li div.flexch-infoz a';
+        this.queryChaptersTitle = 'span';
```

Comment:
How about setting `this.queryChaptersTitleBloat = 'span.date'` to remove the date from the chapter title?
This may avoid re-implementation of the `_getChapters()` method.

Reply:
I modified that code in this new commit for accomplishing the tasks of getting rid of the date field more shortly, as advised.